outplacement/models/outplacement.py

```diff
@@ -185,49 +184,48 @@ def _track_template(self, tracking):
-                                                             doc_names=None)
-        res = {app.id: aliases.get(app.job_id.id) for app in self}
-        leftover = self.filtered(lambda rec: not rec.job_id)
-        if leftover:
-            res.update(super(Outplacement, leftover)._notify_get_reply_to(
-                default=default,
-                records=None,
-                company=company,
-                doc_names=doc_names))
-        return res
+    # This crashes, who uses this?, can it be removed?
+    # @api.multi
+    # def _notify_get_reply_to(
+    #         self, default=None, records=None, company=None, doc_names=None):
+    #     """
+    #     Override to set alias of Outplacements to their job definition
+    #     if any.
+    #     """
+    #     aliases = self.mapped('job_ids')._notify_get_reply_to(default=default,
+    #                                                          records=None,
+    #                                                          company=company,
+    #                                                          doc_names=None)
+    #     res = {app.id: aliases.get(app.job_id.id) for app in self}
+    #     leftover = self.filtered(lambda rec: not rec.job_id)
+    #     if leftover:
+    #         res.update(super(Outplacement, leftover)._notify_get_reply_to(
+    #             default=default, records=None,
+    #             company=company, doc_names=doc_names))
+    #     return res
 
-    @api.multi
-    def message_get_suggested_recipients(self):
-        recipients = super(
-            Outplacement, self).message_get_suggested_recipients()  # noqa: F823,E501 (Odoomagic, Outplacement do exist.)
-        for Outplacement in self:
-            if Outplacement.partner_id:
-                Outplacement._message_add_suggested_recipient(
-                    recipients,
-                    partner=Outplacement.partner_id,
-                    reason=_('Contact'))
-            elif Outplacement.email_from:
-                Outplacement._message_add_suggested_recipient(
-                    recipients,
-                    email=Outplacement.email_from,
-                    reason=_('Contact Email'))
-        return recipients
+    # @api.multi
+    # def message_get_suggested_recipients(self):
+    #     recipients = super(
+    #         Outplacement, self).message_get_suggested_recipients()
+    #     for outplacement in self:
+    #         if outplacement.partner_id:
+    #             outplacement._message_add_suggested_recipient(
+    #                 recipients,
+    #                 partner=outplacement.partner_id,
+    #                 reason=_('Contact'))
+    #         elif outplacement.email_from:
+    #             outplacement._message_add_suggested_recipient(
+    #                 recipients,
+    #                 email=outplacement.email_from,
+    #                 reason=_('Contact Email'))
+    #     return recipients
```

Comment:
Please give a reason why this is commented or remove it.

Reply:
Same as for the whole block of comments, but I'll add a comment on this function as well